syftlib/src/main/java/org/openmined/syft/proto/SyftModel.kt

```diff
@@ -7,6 +7,7 @@ import java.io.File
 
 private const val TAG = "SyftModel"
 
+// SyftModel is a model class for neural network weights. kind of like nn.Module.state_dict
```

Comment:
SyftModel would be a public class available to user. So can you write formal java docs for this class? Like function descriptions and class descriptions using @param, @link etc?